### PR TITLE
MNT Remove TODO comments

### PR DESCRIPTION
**Pull request by sabina-talipova**
## Parent issue
- https://github.com/silverstripe/.github/issues/116

### CODE REVIEW

File: src/Services/LDAPService.php [hunk 1]
```diff
@@ -1102,7 +1102,7 @@ public function addLDAPUserToGroup($userDn, $groupDn)
      * Change a members password on the AD. Works with ActiveDirectory compatible services that saves the
      * password in the `unicodePwd` attribute.
      *
-     * @todo Use the Laminas\Ldap\Attribute::setPassword functionality to create a password in
+     * Use the Laminas\Ldap\Attribute::setPassword functionality to create a password in
```
Comment (sabina-talipova) [inline src/Services/LDAPService.php]: Leave this as a additional comment

File: src/Iterators/LDAPIterator.php
Comment (sabina-talipova) [inline src/Iterators/LDAPIterator.php:1]: Open ticket:
- https://github.com/silverstripe/silverstripe-ldap/issues/62

File: src/Services/LDAPService.php [hunk 2]
```diff
@@ -975,7 +975,7 @@ public function updateLDAPFromMember(Member $member)
             foreach ($member->config()->ldap_field_mappings as $attribute => $field) {
                 $relationClass = $member->getRelationClass($field);
                 if ($relationClass) {
-                    // todo no support for writing back relations yet.
+                    continue;
```
Comment (sabina-talipova) [inline src/Services/LDAPService.php]: Open ticket:
- https://github.com/silverstripe/silverstripe-ldap/issues/62

File: src/Services/LDAPService.php [hunk 3]
```diff
@@ -975,7 +975,7 @@ public function updateLDAPFromMember(Member $member)
...
                 } else {
                     $attributes[$attribute] = $member->$field;
                 }
```
Comment (ChloeSartorelli) [inline src/Services/LDAPService.php]: Why are we adding a `continue;` here? It seems like we should just change the condition like so

```suggestion
                if (!$relationClass) {
                    $attributes[$attribute] = $member->$field;
                }
```

File: src/Services/LDAPService.php (hunk 1 above)
Comment (ChloeSartorelli) [reply src/Services/LDAPService.php]: It seems like this is a todo comment saying we _should_ use that, which we don't currently. We should probably delete this comment and create an issue for it.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: src/Services/LDAPService.php (hunk 3 above)
Comment (sabina-talipova) [reply src/Services/LDAPService.php]: Updated

**Comment by sabina-talipova** [comment]
I've created new ticket: 
- https://github.com/silverstripe/silverstripe-ldap/issues/64

**Review by GuySartorelli — APPROVED** [review]
LGTM

**Pull request merged**